src/main/webapp/projects.html

```diff
@@ -3,13 +3,28 @@
   <head>
```

Comment:
Consider removing optional tags, as suggested by the html style sheet: https://google.github.io/styleguide/htmlcssguide.html#HTML_Style_Rules

Reply:
I'll definitely take a look at this, as I see both you and Obi are doing it and it'd probably be best if we stayed consistent. It hurts my heart a bit as it looks quite different from what I'm used to, but I'll look to change it in the future.